AWSLambdaAPI/Controllers/S3BucketController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Amazon.S3.Model;
+using Amazon.S3.Util;
+using Amazon.S3;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NPOI.SS.Formula.Functions;
+using NPOI.SS.UserModel;
+using NPOI.XSSF.UserModel;
+using ExternalEntities.Misc;
+using Microsoft.Extensions.Options;
+using Repository.Interfaces;
+using Amazon.SecretsManager;
+using Amazon;
+using Amazon.SecretsManager.Model;
+
+namespace AWSLambdaAPI.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class S3BucketController : ControllerBase
+    {
+        private string bucketName = "task6.s3bucket";
+
+        private readonly IOptions<DBSettings> _dbSettings;
+        private readonly Repository.Interfaces.IToDoRepository _repo;
+
+        public S3BucketController(IOptions<DBSettings> dbSettings, IToDoRepository repository)
+        {
+            _dbSettings = dbSettings;
+            _repo = repository;
+        }
+
+        [HttpPost]
+        public async Task Post(IFormFile file)
+        {
+            var S3client = new AmazonS3Client();
+
+            bool bucket = await AmazonS3Util.DoesS3BucketExistV2Async(S3client, bucketName);
+            if (!bucket)
+            {
+                var bucketRequest = new PutBucketRequest()
+                {
+                    BucketName = bucketName,
+                    UseClientRegion = true,
+                };
+
+                await S3client.PutBucketAsync(bucketRequest);
+            }
+
+            var objectRequest = new PutObjectRequest()
+            {
+                BucketName = bucketName,
+                Key = file.FileName,
+                //ContentType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
```

Comment:
remove dead / commented code

Reply:
unloaded the project for now as its not needed.